src/App.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Route, Switch, Redirect } from 'react-router-dom';
+
+import './App.scss';
+
+import { Navigation } from './components/Navigation';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { NotFoundPage } from './components/NotFoundPage';
+
+const App = () => (
+  <div className="App">
+    <header>
+      <Navigation />
+    </header>
+
+    <main className="container">
+      <Switch>
+        <Route path="/people">
+          <PeoplePage />
+        </Route>
```

Comment:
Вторая. Я написал так же, но Миша отклонил. В таких случаях лучше Роуту передавать дочерний компонент в пропсах. То есть
```suggestion
         <Route path="/people" component={PeoplePage} />
```